src/pages/CreateRecipe.js

```diff
@@ -62,43 +68,67 @@ export default function Fridge({ addRecipe }) {
     document.querySelector('[name=instruction]').value = ''
     document.querySelector('[name=products]').value = ''
   }
+}
 
-  // function handleSubmit(event) {
-  //   event.preventDefault()
-  //   onSubmit(recipeState)
-  // }
+const InputStyled = styled.input`
+  box-sizing: border-box;
+  width: 100%;
+  display: block;
+  outline: none;
+  font-size: 16px;
+  padding: 0 0 0 5px;
+  background: white;
+  color: gray;
+  font-family: Arial;
+  height: ${props => props.height || 35}px;
+`
 
-  // function handleRecipeChange(event) {
-  //   setRecipeState({
-  //     ...recipeState,
-  //     [event.target.name]: event.target.value,
-  //   })
-  // }
-}
+const SpanStyled = styled.span`
+  background: #f2f2f2;
+  display: block;
+  padding: 7px;
+  width: 100%;
+  text-align: center;
+  color: gray;
+  font-family: Arial;
+  font-size: 16px;
+`
+
+const LabelStyled = styled.label`
+  float: left;
+  box-shadow: 0px -1px 26px -8px rgba(0, 0, 0, 0.2);
+  border: 1px solid #dddddd;
+  border-radius: 4px;
+  margin-top: -27px;
+  margin-left: 17px;
+  background: #ffffff;
+  height: 26px;
+  padding: 3px 5px;
+  color: #b9b9b9;
+  font-size: 14px;
+  font-family: Arial;
+`
+
+const UlStyled = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+`
+const LiStyled = styled.li`
+  margin: 40px 0 5px;
+  background: white;
+  padding: 10px 0 0;
+  border: 1px solid #dddddd;
+  border-radius: 5px;
+`
 
 const CardForm = styled.form`
   margin-top: 70px;
   display: flex;
   flex-direction: column;
   align-items: center;
 `
-const LabelStyled = styled.label`
-  width: 300px;
-  margin-top: 23px;
-`
 
-const InputStyled = styled.textarea`
-  width: 100%;
-  height: ${props => props.height || 45}px;
-  padding: 12px;
-  border: 1px solid #ccc;
-  border-radius: 6px;
-  box-sizing: border-box;
-  font-size: 18px;
-  background: #fffaf4;
-  box-shadow: 0px 5px 16px -3px rgba(0, 0, 0, 0.2);
-  resize: none;
-`
 const ButtonStyled = styled.button`
   width: 270px;
   height: 45px;
```

Comment:
Better is to work with padding here, instead of a fix width and height. If you have a small browserview, the button would be trimmed.